phpBB/styles/prosilver/template/ucp_pm_viewfolder.html

```diff
@@ -1,130 +1,283 @@
+									{% for recipient in to_recipient %}
+									<li class="pm-recipient">
+										<i class="fa fa-user fa-fw" aria-hidden="true"></i>
+										{% if recipient.IS_GROUP %}<a href="{recipient.U_VIEW}">{{ recipient.NAME }}</a>{% else %}{{ recipient.NAME_FULL }}{% endif %}
+									</li>
+									{% endfor %}
+								</ul>
+							</div>
+							{% endif %}
+							{% if bcc_recipient %}
+							<div class="pm-recipient-bcc">
+								{{ lang('BCC') }}{{ lang('COLON') }}
+								<ul class="pm-recipient-list">
+									{% for recipient in bcc_recipient %}
+									<li class="pm-recipient">
+										<i class="fa fa-user fa-fw" aria-hidden="true"></i>
+										{% if bcc_recipient.IS_GROUP %}<a class="pm-recipient-link" href="{{ recipient.U_VIEW }}">{{ recipient.NAME }}</a>{% else %}{{ recipient.NAME_FULL }}{% endif %}
+									</li>
+									{% endfor %}
+								</ul>
+							</div>
+							{% endif %}
+						</dd>
+					</div>
+				</div>
+			</div>
+		</div>
+		{% endif %}
+		{% EVENT ucp_pm_history_review_before %}
+		<div class="pm-message">
+			<!--Start displaying complete messages-->
+			{% for history in history_row %}
+				<ul class="pm-message-list{% if not history.S_IS_SELF %} pm-not-self{% endif %}">
+					<li class="pm-message-item ">
+						<span class="pm-message-primary">
+							<span class="pm-message-primary-item">
+								<div class="pm-message-author">
+									<i class="fa fa-user fa-fw" title="{{ message.MESSAGE_AUTHOR }}" aria-label="{{ message.MESSAGE_AUTHOR }}"></i>
+									{{ history.MESSAGE_AUTHOR_FULL }}
+								</div>
+								<div class="pm-message-date">
+									<i class="fa fa-clock-o fa-fw" title="{{ message.SENT_TIME }}" aria-label="{{ message.SENT_TIME }}"></i>
+									{{ history.SENT_DATE }}
+									<!--<p id="pm-time"></p>-->
+								</div>
+							</span>
+							<span class="pm-message-secondary">
+								{% if history.S_CURRENT_MSG %}
+									<!--Some way to indicate this is the message clicked, change of color isn't good enough-->
+									<!--<i class="fa fa-certificate fa-fw msg-unread" title="Unread message" aria-label="Unread message"></i>-->
+								{% endif %}
+								<a class="pm-message-title{% if history.S_CURRENT_MSG %} pm-message-unread{% endif %}" href="{{ history.U_VIEW_MESSAGE }}" >{{ history.SUBJECT }}</a>
+								<br>
+								{% if history.MESSAGE %}
+									{{ history.MESSAGE }}
+								{% else %}
+									<span class="error">{{ lang('MESSAGE_REMOVED_FROM_OUTBOX') }}</span>
+								{% endif %}
+							</span>
+						</span>
+					</li>
+				</ul>
+			{% endfor %}
+		</div>
+		{% EVENT ucp_pm_history_review_before %}
+		<!--Compose box, BBCode buttons, submit/preview buttons-->
+		<div class="pm-compose">
+			<div class="pm-subject-box">
+				<input class="pm-inputbox" id="subject" name="subject" maxlength="{% if S_NEW_MESSAGE %}120{% else %}124{% endif %}" tabindex="2" type="text" placeholder="{{ lang('PM_TITLE') }}" >
```

Comment:
Remove the space: `" >`. The `maxlength` value should be calculated in the backend.